### ISSUE TITLE
Unexpected behaviors after key mapping

### ISSUE THREAD

**Issue by joshua7v** [asvetliakov/vscode-neovim#50]
Hi, thank you for putting effort into vscode - neovim integration
really like it

but after some tastes, I ran into an issue when i need to remap keys
I mapped some key strokes to call `editor.action.formatDocument`,
editor called the command but also with some weird monifications.
I've recorded gifs below.
But when I manually call the command in nvim command prompt, it does not have this issue.
![test2](https://user-images.githubusercontent.com/4159858/69476349-e0fbfe80-0e13-11ea-8170-f30c6bf01035.gif)
![test1](https://user-images.githubusercontent.com/4159858/69476350-e35e5880-0e13-11ea-94c8-090889308ceb.gif)

and here is my mapping

```vml
nnoremap <space><space> :<c-u>call VSCodeCall('editor.action.formatDocument')<cr>
vnoremap <space><space> :<c-u>call VSCodeCall('editor.action.formatSelection')<cr> 
```

**Comment by asvetliakov** [comment]
weird issue
this works
```
nnoremap <expr> <space><space> VSCodeCall('editor.action.formatDocument')
```

also i recommend to use VSCodeNotify unless you really need blocking call (e.g. you calling few commands and need to wait for previous one)

**Comment by joshua7v** [comment]
ok, thanks. Now it works like a charm.
yeah, I tried `VSCodeNotify` first, it has the same issue.

**Comment by asvetliakov** [comment]
Debugged the problem - the problem it's the vscode command line sends additional "<C-u>" on change/initial appearing (to re-enter text) and it's causing weird behavior. ```<expr>``` doesn't use command line mode, so it works fine. Will fix today, thanks for reporting

**Comment by asvetliakov** [comment]
The fix is landed in ```0.0.32```
